roaster-tm4j/roaster-tm4j-common/src/test/java/hu/icellmobilsoft/roaster/tm4j/common/DefaultTm4jReporterTest.java

```diff
@@ -0,0 +1,225 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.tm4j.common;
+
+import hu.icellmobilsoft.roaster.tm4j.common.api.TestCaseId;
+import hu.icellmobilsoft.roaster.tm4j.common.client.Tm4jService;
+import hu.icellmobilsoft.roaster.tm4j.common.client.model.Execution;
+import hu.icellmobilsoft.roaster.tm4j.common.config.InvalidConfigException;
+import hu.icellmobilsoft.roaster.tm4j.common.config.Tm4jReporterConfig;
+import hu.icellmobilsoft.roaster.tm4j.common.api.TestCaseData;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.ArgumentCaptor;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class DefaultTm4jReporterTest {
```

Comment:
javadoc minden osztalyra meg ha teszt is az

Reply:
Miért? A teszt javadoc-ot nem generáljuk ki.

Reply:
A teszteknel a javadocnak nincs jelentosege, de arra hasznos hogy tudjuk nagyjabol ki alkotta es melyik verzioba keszult, ugy az ember kepet kap mar hogy aljon hozza

Reply:
Erről a git blame sokkal pontosabb képet tud adni.